class U
{
	int test()
	{
		try
		{
		}
		catch (ArithmeticException ex)
		{
			return 10;
		}
	}
}
